src/backend/api/meals.js

```diff
@@ -2,6 +2,7 @@ const express = require("express");
 const { prependListener } = require("../database");
 const router = express.Router();
 const knex = require("../database");
+
 //Returns all meals
 router.get("/", async (request, response) => {
```

Comment:
It doesn't really work if you have defined the same route -> .get("/") twice!
Nodejs doesn't know which function should be executed, even if in one case you're sending in query parameters and in the other not.
Try to merge them!

Reply:
done